cli-feed-v3.json

```diff
@@ -404,6 +404,45 @@
         }
       ]
     },
+    "2.11.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.23",
+      "cli": "https://functionscdn.azureedge.net/public/2.2.1/Azure.Functions.Cli.win-x86.2.2.1.zip",
+      "nodeVersion": "8.11.1",
+      "localEntryPoint": "func.exe",
+      "itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-10300",
+      "projectTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.AzureFunctions.ProjectTemplates/2.0.0-10300",
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/2.0.0-10288.zip",
+      "sha2": "DB1A6D8A20E600D57824F03EC1431D2169C26A952D6BE2FC815EAC35CEC4EE0D",
+      "FUNCTIONS_EXTENSION_VERSION": "~2",
+      "requiredRuntime": ".NET Core",
+      "minimumRuntimeVersion": "2.1",
+      "standaloneCli": [
+        {
+          "OperatingSystem": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.1.748/Azure.Functions.Cli.osx-x64.2.2.1.zip",

```

Comment:
Download link is incorrect
https://functionscdn.azureedge.net/public/2.2.1/Azure.Functions.Cli.osx-x64.2.2.1.zip

Reply:
For all versions of standalone cli's

Reply:
Argh. Yeah I'll fix it.